models/src_google_play.yml

```diff
@@ -7,7 +7,7 @@ sources:
     loader: Fivetran
     loaded_at_field: _fivetran_synced
 
-    freshness:
+    freshness: 
```

Comment:
Just wanted to check that we do in fact want this freshness range to be applied to all of the source tables.

Reply:
that's a good point -- given the data lags we've seen (and how they're pretty inconsistent across tables) i am wondering if we should remove the freshness tests (or at least just have warnings instead of errors?)

Reply:
For Apple I suggested we have a 48 hour warn window and a week error. Let's do the same here.